rewards/calc_rewards.py

```diff
@@ -189,9 +190,12 @@ def approve_root(
             )
             if chain not in NO_BOOST_CHAINS:
                 upload_boosts(boosts, chain)
+            claimable_rewards_data = get_claimable_rewards_data(chain, end)
```

Comment:
if change title of function this could be claimable_rewards_deficits and same in L195